### DISCUSSION TITLE
webdav很有用，请不要移除它

### DISCUSSION THREAD

**Discussion by yeweimei** [Richasy/Bili.Copilot#937] [Q&A]
RT，我通过这个来看alist上面的视频，感觉非常好用，最近在新版本里发现这个功能被标记成了“预览（可能会在未来移除）”，求求了，这个功能很有用，可以不要移除吗，谢谢作者大大。

**Comment by Richasy** [comment]
不必担心，在可预见的未来不会移除这个功能。添加这个提示只是提醒你不要过分依赖它，因为它只是在现有播放组件上的一个尝试，不会为它实现完整的功能（比如进度记录等）
